packages/googlesheets/test/index.js

```diff
@@ -68,3 +68,48 @@ describe('batchUpdateValues', () =>{
     expect(result).to.eql(state);
   });
 });
+
+describe('fn function', () => {
```

Comment:
these mock tests don't look right either :thinking: 

You'll need to call `execute` to trigger the creation of the sheets client, and then call `fn(callback)`. Your callback needs to ensure that a sheets client is passe as the second argument (by the adaptor, not by your test code)